src/Compilation.zig

```diff
@@ -3475,17 +3475,43 @@ fn reportRetryableAstGenError(
         },
     };
 
-    const err_msg = if (file.pkg.root_src_directory.path) |dir_path|
-        try Module.ErrorMsg.create(
-            gpa,
-            src_loc,
-            "unable to load '{s}" ++ std.fs.path.sep_str ++ "{s}': {s}",
-            .{ dir_path, file.sub_file_path, @errorName(err) },
-        )
-    else
-        try Module.ErrorMsg.create(gpa, src_loc, "unable to load '{s}': {s}", .{
-            file.sub_file_path, @errorName(err),
-        });
+    const err_msg = switch (err) {
+        error.ImportPathMismatch => blk: {
```

Comment:
I haven't touched this part of the code in a while but to me it seems like this belongs in `Sema.zirImport`.

Reply:
I have not moved this code. The scope of the function seems correct, as it is supposed to report this kind of error.

Should I change anything here?